### ISSUE TITLE
[Request] EfficientNet as a encoder

### ISSUE THREAD

**Issue by crossknight** [qubvel/segmentation_models.pytorch#19]
First, I would like to thank you for making a great project.
I found that you added EfficientNet to your Keras project and I was wondering if you could add that to this Pytorch project too?

**Comment by qubvel** [comment]
Hi @crossknight 
I will consider this option, but honestly speaking efficientnets in my experiments achive worse quality, while takes more GPU memory during training (compared with se-resnext50/resnet34)

**Comment by crossknight** [comment]
@qubvel Hmm, I think it's a state-of-the-art model in classification task for now. 
Is its accuracy worse than se-resnext50/resnet34?

**Comment by qubvel** [comment]
In classification task - yes. But it is not guaranteed that it will be the best in other CV tasks. It is harder to train and it requires more GPU memory (google use tpu for training, but I am limited with GPU resources). Moreover, maybe one of the reason why efficientnet shows so high results is higher resolution of input images.
But anyway it is better to check it for your specific data, may be you will find a way to get better results.

**Comment by crossknight** [comment]
I see...
Good to know. Thank you.
